Neos.Neos/Classes/Command/SiteCommandController.php

```diff
@@ -217,6 +217,17 @@ public function importCommand($packageKey = null, $filename = null)
             $this->outputLine('You have to specify either "--package-key" or "--filename"');
             $this->quit(1);
         }
+
+        // Since this command uses a lot of memory when large sites are imported, we warn the user to watch for
+        // the confirmation of a successful import.
+        $this->outputLine('<b>This command can use a lot of memory when importing sites with many resources.</b>');
+        $this->outputLine('If the import is successful, you will see a message saying "Import of site ... finished".');
+        $this->outputLine('If you do not see this message, the import failed due to insufficient memory.');
```

Comment:
Is this really the only reason that the import can fail (silently)? If not, we should make the explanation less absolute, e.g. `the import most likely failed due to...`

Reply:
Good point, @albe

Reply:
You're right, there's probably other reasons for it to fail. I'll change the text a bit.